src/Workspaces/Core/Portable/Workspace/Solution/ProjectInfo.cs

```diff
@@ -598,18 +657,19 @@ public static ProjectAttributes ReadFrom(ObjectReader reader)
                 return new ProjectAttributes(
                     projectId,
                     VersionStamp.Create(),
-                    name,
-                    assemblyName,
-                    language,
-                    filePath,
-                    outputFilePath,
-                    outputRefFilePath,
+                    name: name,
+                    assemblyName: assemblyName,
+                    language: language,
                     compilationOutputFilePaths,
```

Comment:
Thank you for using named arguments here to reduce the risk of crossing the parameters since there's a lot strings.